Scripts/Packaging/build_packages.sh

```diff
@@ -23,19 +23,101 @@
 
 set -e
 
+# Default values
 GIT_TOP_LEVEL=$(git rev-parse --show-toplevel)
+ROCM_EXAMPLES_ROOT="$GIT_TOP_LEVEL"
+PACKAGE_NAME="ROCm-SDK-Examples"
+PACKAGE_VERSION="6.2.0"
+PACKAGE_INSTALL_PREFIX="/opt/rocm/examples"
+TEST_PACKAGE_INSTALL_PREFIX="/opt/rocm/examples-test"
+BUILD_DIR="$ROCM_EXAMPLES_ROOT/build"
+DEB_DIR="$BUILD_DIR/deb"
+RPM_DIR="$BUILD_DIR/rpm"
+DEB_PACKAGE_RELEASE="local.9999"
+RPM_PACKAGE_RELEASE="local.9999"
+CPACKGEN=""
 
-# Inputs and Defaults
-ROCM_EXAMPLES_ROOT="${1:-$GIT_TOP_LEVEL}"
-PACKAGE_NAME="${2:-ROCm-SDK-Examples}"
-PACKAGE_VERSION="${3:-6.2.0}"
-PACKAGE_INSTALL_PREFIX="${4:-/opt/rocm/examples}"
-TEST_PACKAGE_INSTALL_PREFIX="${5:-/opt/rocm/examples-test}"
-BUILD_DIR="${6:-$ROCM_EXAMPLES_ROOT/build}"
-DEB_DIR="${7:-$BUILD_DIR/deb}"
-RPM_DIR="${8:-$BUILD_DIR/rpm}"
-DEB_PACKAGE_RELEASE="${9:-local.9999}"
-RPM_PACKAGE_RELEASE="${10:-local.9999}"
+PACKAGE_CONTACT="ROCm Developer Support <rocm-dev.support@amd.com>"
+PACKAGE_DESCRIPTION_SUMMARY="A collection of examples for the ROCm software stack"
+PACKAGE_HOMEPAGE_URL="https://github.com/ROCm/ROCm-examples"
+
+# Getopt argument parsing
+VALID_ARGS=$(getopt -o hcr --long help,clean,release,root:,pkgname:,version:,install-prefix:,test-install-prefix:,build-dir:,deb-dir:,rpm-dir:,deb-release:,rpm-release:,cpackgen: -- "$@")
+if [[ $? -ne 0 ]]; then
+    echo "Invalid arguments"
+    exit 1
+fi
+
+eval set -- "$VALID_ARGS"
+
+while [ : ]; do
+  case "$1" in
+    -h | --help)
+      echo "Usage: build_rocm_examples.sh [options]"
+      echo "Options:"
+      echo "  --root <path>                    Set the root of the ROCm examples"
```

Comment:
```suggestion
      echo "  --src-dir<path>                    Set the source directory"
```
Changing to `src-dir` to match other path related options. Maybe we should move it lower to also group them all together.